examples/paragraph/backend/Cargo.toml

```diff
@@ -11,3 +11,4 @@ edition = "2021"
 
 [dependencies]
 moon = { path = "../../../crates/moon" }
+lang = { path = "../../../crates/lang"}
```

Comment:
~Please move it to Moon's dependencies. We can make it optional later to make sure long `Lang` enum won't slow down the compiler.~ Then remove the line.